autopts/wid/tmap.py

```diff
@@ -562,6 +563,7 @@ def hdl_wid_506(params: WIDParams):
     source_id = 0x00
     btp.cap_broadcast_adv_stop(source_id)
     btp.cap_broadcast_source_stop(source_id)
+    time.sleep(10) # Allow some time for the stack to stop the broadcast
```

Comment:
The right solution would be to either implement an BTP event, or for the IUT to not send the response before the BIG is fully stopped. We did something similar for a BAP test. 

Adding a sleep in the AutoPTS code is, IMO, not the right solution.

Reply:
The PR that fixed this for BAP: https://github.com/zephyrproject-rtos/zephyr/pull/79329

Reply:
Agree, see https://github.com/zephyrproject-rtos/zephyr/pull/86625 instead